### ISSUE TITLE
Improve Readme

### ISSUE THREAD

**Issue by louisefindlay23** [MLH-Fellowship/pod-3.1.3-portfolio#18]
Improve Readme (similar to #15).

Could add another section showing new features we've added (once PRs have merged) and add a screenshot of the current portfolio website.

Possibly if the readme gets big, move the developer info into a seperate docs folder?

**Comment by Primata** [comment]
I think I'd like to take this one. @louisefindlay23 what do you mean by developer info? _data section where it explains on how to setup your profile?

**Comment by louisefindlay23** [comment]
Cool. Go for it, @SnowPrimate. 

By developer info, I meant the existing Readme info or even just the development section so it's geared more for either people wanting to add data or run the website locally rather than introducing what the website is about and its features that a non-technical, lay-person would want to know.

**Comment by Primata** [comment]
Waiting for issues #7, #21, and any other implementation we might have for a screenshot of the portfolio template.
I've noticed Google Analytics has already been added by another pull request.
Added links, installation process, and improved readability. The first paragraph was incomplete.

**Comment by kronaemmanuel** [comment]
Another thing that people might want (I certainly did) is [live reload](https://jekyllrb.com/news/2018/01/02/jekyll-3-7-0-released/). So when they make a change, they don't have to refresh in their browser. For that we can use the `--livereload` flag. So instead of:
```
bundler exec jekyll serve
```
We should have:
```
bundler exec jekyll serve --livereload
```

**Comment by louisefindlay23** [comment]
I had to put `--force-polling` for WSL2 for reloading. That's just for the server, not the browser.

**Comment by Primata** [comment]
Waiting on PRs #77 #81  and #85
